devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -64,8 +64,10 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0019", manufacturer: "Aurora", model: "WallDimmerMaster", deviceJoinName: "Aurora Smart Rotary Dimmer"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E1A-AC2", deviceJoinName: "Sengled DownLight"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "MLI", model: "ZBT-DimmableLight", deviceJoinName: "Müller Licht Tint Bulb Dimming"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: "LIGHTIFY Edge-lit flushmount", deviceJoinName: "SYLVANIA SMART+ Flush Mount"
```

Comment:
Can you leave this line as it was?

Reply:
Hi @tpmanley ,

Line 68 is exactly same as the red row above.
please check this if it is correct

Reply:
The indentation is different. It can be tough to get the indentation right in the IDE but if you copy the indentation from line 66 and then paste it over the indentation on line 68 it should put it back to two tabs

Reply:
@tpmanley please check if I did correctly.
Thanks!

Reply:
Looks good now!